recommendations.md

```diff
@@ -1 +1,33 @@
-#Recommendations
+# Recommendations
+No existing solutions for the problem were found. Based on the conclusions, a description is given of what an ideal solution looks like. This solution is presented in the form of a platform.
+
+## The ideal platform
+The platform converts resources, mainly in the form of time and knowledge, into an extension to the current FeedbackFruits platform. The goal of this extension should be to fulfill a particular need. A goal can be identified as the answer to finishing the phrase: "With FeedbackFruits I would like to ..". Each goal can be devided in challlenges, sub-goals that need to be met in order to meet the goal. If these challlenges are to large to be solved by one or two pioneers, then the challlenges must be devided into sub-challenges in such a way that they can be solved by one or two pioneers.
+
+## Resources
+The platform facilitates a structured process to reach a goal. On a high level, this process consists of two parts: the gathering and spending of resources. The most important resources are time and knowledge. Less important resources can include money, servers and licensed materials. Resources can be gathered by creating a hype around the goal. Once enough resources have been acquired a team of pioneers can start implementing an extenions to achieve the goal. This will result in resources being spent.
+
+## Pioneers
+Pioneers can be identified according to the resources they contribute. Pioneers can fall into one of three categories:
+- Core team (time and knowledge)
+- Contributors (knowledge)
+- Backers (other resources)
+
+The core team are a select group of pioneers that take the responsibility for a project. Dividing the goal into challlenges and sub-challenges is the job of the core members, thus they will provide the structure used for the software development process. Contributors can pick up one or multiple of these sub-challenges and implement the goal of those sub-challlenges. Reviewing code or designs also is a contribution to a sub-challenge. Backers are pioneers that try to improve the project without implementing or reviewing solutions for a (sub-)challlenge. This can be done with domain-specific material (like learning methods e.g.) or more generic support (like money e.g.).
```

Comment:
`domain-specific material (like learning methods e.g.) or more generic support (like money e.g.).`

There is no need to split this into these two options. I would make this a more generic list of possible contributions from backers